docs/en_US/backup_dialog.rst

```diff
@@ -251,6 +251,16 @@ tab to provide other backup options.
      table locks at the beginning of the dump. Instead, fail if unable to lock a
      table within the specified timeout.
 
+Click the *Objects* tab to continue.
+
+.. image:: images/backup_object_selection.png
+    :alt: Select objects in backup dialog
+    :align: center
+
+* Select the objects from tree to take backup of selected objects only.
+    * If selected schema then it will take backup of that selected schema only.
```

Comment:
If Schema is selected then it will take the backup of that selected schema only.